src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/store/application.ts

```diff
@@ -276,14 +276,12 @@ export const useApplicationStore = defineStore("application", {
 
       // Character References step data
       if (
-        wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id]?.[0]?.firstName &&
         wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id]?.[0]?.lastName &&
         wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id]?.[0]?.emailAddress
       ) {
         this.draftApplication.characterReferences =
           wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id];
       } else if (
-        wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id]?.[0]?.firstName === "" &&
         wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id]?.[0]?.lastName === "" &&
         wizardStore.wizardData[wizardStore.wizardConfig.steps.characterReferences.form.inputs.characterReferences.id]?.[0]?.emailAddress === ""
       ) {
```

Comment:
this might be a bit janky now. I read this as if we blank out lastName and emailAddress that we will wipe out everything. Even if we have first name. 

But ... technically lastName and emailAddress should always be required.